packages/react-component-library/src/components/Field/Field.tsx

```diff
@@ -7,7 +7,7 @@ import { StyledHintText } from './partials/StyledHintText'
 import { getId } from '../../helpers'
 
 export type ErrorType = {
-  error: boolean | string
+  error: boolean | string | null | undefined
```

Comment:
`undefined` was coming from `meta.error` in the Formik example – I think it's reasonable to allow `null` and `undefined` as they could be being used if errors are being held as strings.